web/src/app/schedules/temp-sched/TempSchedAddShiftsStep.tsx

```diff
@@ -46,6 +60,10 @@ type AddShiftsStepProps = {
 
   scheduleID: string
   edit?: boolean
+
+  showCoverageGapsWarning?: boolean
```

Comment:
rather than being dependent on a prop to show the warning, can the list determine this itself? looks like it only needs a call to `getCoverageGapItems` from `'./shiftsListUtil'`

Reply:
Yep with the props I think the child could determine for itself if there's gaps. It just needs the shifts and start and end. I like that change because it avoids having repeat data in the props for `tempSchedAddShiftsStep` when it can easily be derived.

Reply:
Just submitted a new version. Let me know if that resolves this.